### PR TITLE
New pages for marketing website

### PR DESCRIPTION
**Pull request by jeromeZestyIO**
Page links on dev.

https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/solutions/headless-cms/headless-cms-benefits/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/solutions/headless-cms/enterprise-headless-cms/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/solutions/headless-cms/digital-asset-management/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/solutions/headless-cms/commerce/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/solutions/headless-cms/headless-mobile/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/solutions/headless-cms/blogs-editorial/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/features/localization/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/features/cdn-hosting/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/features/digital-governance/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/features/elastic-scalability/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/features/headless-cms/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/features/multi-language/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/features/seo/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/features/multi-site-management/
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/agency-partners/partner-program-directory/

### CODE REVIEW

File: src/blocks/zesty/TabContent/TabContent.js.js [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+                            e.img?.data[0].url ||
+                            FillerContent.logos[0].url
+                          }
+                          alt=""
+                          component="img"
+                          sx={{
+                            width: '100%',
+                          }}
+                        />
+                         {/* <ZestyImage
```
Comment (darwin808) [inline src/blocks/zesty/TabContent/TabContent.js.js]: remove comments

File: src/views/zesty/DxpBenefit.js [hunk 2]
```diff
@@ -64,28 +64,250 @@
...
+    header: content.what_is,
+    mainImage: content.what_is_graphic?.data[0]?.url,
+    backgroundColor: theme.palette.zesty.zestyWhite,
+    marginTop: 5,
+  };
+
+  const alternateColumnsData = [
+    {
+      content: content.benefit_1,
+      image: content.benefit_1_graphic?.data[0].url,
```
Comment (darwin808) [inline src/views/zesty/DxpBenefit.js]: add optional chaining

File: src/views/zesty/DxpBenefit.js [hunk 3]
```diff
@@ -64,28 +64,250 @@
...
+      ).replace(/\<\/h2> *\<\/h2>/g, '</h2>');
+    }
+
+    return paragraph;
+  };
+
+  const whyZestyData = [
+    {
+      content: splitParagraph(content.why_zesty, 0, 1),
+      image: content.why_zesty_graphic?.data[0].url,
```
Comment (darwin808) [inline src/views/zesty/DxpBenefit.js]: optional chaining

File: src/views/zesty/HeadlessBlogProductPage.js [hunk 4]
```diff
@@ -38,28 +38,100 @@
...
+  };
+
+  const howZestyData = [
+    {
+      content: content.how_1,
+      icon_image: content.how_1_graphic?.data[0].url,
+    },
+    {
+      content: content.how_2,
+      icon_image: content.how_2_graphic?.data[0].url,
```
Comment (darwin808) [inline src/views/zesty/HeadlessBlogProductPage.js]: optional chaining

File: src/views/zesty/HeadlessCmsFeature.js [hunk 5]
```diff
@@ -22,28 +22,129 @@
...
+  };
+
+  const alternateColumnsData = [
+    {
+      content: content.what_section_1,
+      image: content.what_section_1_graphic?.data[0].url,
+    },
+    {
+      content: content.what_section_2,
+      image: content.what_section_2_graphic?.data[0].url,
```
Comment (darwin808) [inline src/views/zesty/HeadlessCmsFeature.js]: optional chaining

File: src/views/zesty/HeadlessCmsMultiLang.js [hunk 6]
```diff
@@ -39,28 +39,103 @@
...
+    {
+      content: content.solution_1,
+      image: content.solution_1_graphic?.data[0].url,
+    },
+    {
+      content: content.solution_2,
+      image: content.solution_2_graphic?.data[0].url,
+    },
+    {
+      content: content.solution_3,
```
Comment (darwin808) [inline src/views/zesty/HeadlessCmsMultiLang.js]: optional chaining

File: src/views/zesty/HeadlessLocalization.js [hunk 7]
```diff
@@ -39,28 +39,105 @@
...
+      content: content.how_1,
+      image: content.how_1_graphic?.data[0].url,
+    },
+    {
+      content: content.how_2,
+      image: content.how_2_graphic?.data[0].url,
+    },
+    {
+      content: content.how_3,
+      image: content.how_3_graphic?.data[0].url,
```
Comment (darwin808) [inline src/views/zesty/HeadlessLocalization.js]: optional chaining

File: src/views/zesty/HeadlessMobile.js [hunk 8]
```diff
@@ -41,28 +41,92 @@
...
+      content: content.benefit_1,
+      icon_image: content.benefit_1_graphic?.data[0].url,
+    },
+
+    {
+      content: content.benefit_2,
+      icon_image: content.benefit_2_graphic?.data[0].url,
+    },
+
+    {
```
Comment (darwin808) [inline src/views/zesty/HeadlessMobile.js:79]: optional chaining

File: src/views/zesty/HeadlessMultiSite.js [hunk 9]
```diff
@@ -51,28 +51,154 @@
...
+  const whatIsMultiSiteProps = {
+    content: content.what_is_multi_site,
+  };
+
+  const whyChooseData = [
+    {
+      title: content.why_1,
+      icon_image: content.why_1_graphic?.data[0].url,
+    },
+    {
```
Comment (darwin808) [inline src/views/zesty/HeadlessMultiSite.js:96]: optional chaining

File: src/views/zesty/ScalableCm.js [hunk 10]
```diff
@@ -37,28 +37,94 @@
...
+  };
+
+  const alternateColumnsData = [
+    {
+      content: content.benefit_1,
+      image: content.benefit_1_graphic?.data[0].url,
+    },
+    {
+      content: content.benefit_2,
+      image: content.benefit_2_graphic?.data[0].url,
```
Comment (darwin808) [inline src/views/zesty/ScalableCm.js]: optional chaining

**Review by darwin808 — CHANGES_REQUESTED** [review]
please see comment

**Comment by jeromeZestyIO** [comment]
Added optional chaining.

**Review by darwin808 — APPROVED** [review]

**Pull request merged**